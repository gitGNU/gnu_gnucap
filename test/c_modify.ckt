' a test of the fault and modify commands
V1 1 0 1
R1 1 2 1
R2 2 0 1
.option trace
.print op v(nodes)
.op
.fault R1=10
.op
.modify R2=10
.op
.fault R2=100
.op
.fault V1=.001
.op
.list
.unfault
.list
.op
.end
