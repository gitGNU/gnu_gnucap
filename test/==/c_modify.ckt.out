' a test of the fault and modify commands 
print op v(nodes) 
op 
#           v(1)       v(2)      
 300.15     1.         0.5       
fault R1=10 
op 
#           v(1)       v(2)      
 300.15     1.         0.090909  
modify R2=10 
op 
#           v(1)       v(2)      
 300.15     1.         0.5       
fault R2=100 
op 
#           v(1)       v(2)      
 300.15     1.         0.90909   
fault V1=.001 
op 
#           v(1)       v(2)      
 300.15     0.001      909.09u   
list 
V1  1  0  0.001
R1  1  2  10.
R2  2  0  100.
.option trace 
.print op v(nodes) 
.op 
.fault R1=10 
.op 
.modify R2=10 
.op 
.fault R2=100 
.op 
.fault V1=.001 
.op 
unfault 
list 
V1  1  0  DC   1.
R1  1  2  1.
R2  2  0  10.
.option trace 
.print op v(nodes) 
.op 
.fault R1=10 
.op 
.modify R2=10 
.op 
.fault R2=100 
.op 
.fault V1=.001 
.op 
.list 
.unfault 
op 
#           v(1)       v(2)      
 300.15     1.         0.90909   
end 
